src/main/java/com/taskviewer/api/web/security/jwt/JwtUserDetailsFactory.java

```diff
@@ -0,0 +1,21 @@
+package com.taskviewer.api.web.security.jwt;
+
+import com.taskviewer.api.model.User;
+import io.jsonwebtoken.Claims;
+
+import java.util.Collections;
+
+public class JwtUserDetailsFactory {
+
+  public static JwtUserDetails create(User user) {
+    return new JwtUserDetails(user);
+  }
+
+  public static JwtUserDetails create(Claims claims) {
```

Comment:
I mean this class will encapsulates the `user`, not pass it as a param, but encapsulates.
and you will get rid of these static methods